__tests__/formats/stylusVariables.test.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+var formats = require('../../lib/common/formats');
+var stylus = require('stylus');
+
+var file = {
+  "destination": "__output/",
+  "format": "stylus/variables",
+  "name": "foo"
+};
+
+var propertyName = "color-base-red-400";
+var propertyValue = "#EF5350";
+
+var dictionary = {
+  "allProperties": [{
+    "name": propertyName,
+    "value": propertyValue,
+    "original": {
+      "value": propertyValue
+    },
+    "attributes": {
+      "category": "color",
+      "type": "base",
+      "item": "red",
+      "subitem": "400"
+    },
+    "path": [
+      "color",
+      "base",
+      "red",
+      "400"
+    ]
+  }]
+};
+
+var formatter = formats['stylus/variables'].bind(file);
+
+describe('formats', () => {
+  describe('stylus/variables', () => {
+
+    it('should have a valid stylus syntax', done => {
+      stylus.render(formatter(dictionary),
+        function (err, css) {
+          if (err) {
+            return done(new Error(err));
+          }
+          expect(css).toBeDefined();
+          return done();
+        });
+    });
+
+  });
+});
```

Comment:
@dbanksdesign  this tests is pretty similar to the [lessVariables.test.js](https://github.com/amzn/style-dictionary/blob/main/__tests__/formats/lessVariables.test.js) or the [scssVariables.test.js](https://github.com/amzn/style-dictionary/blob/main/__tests__/formats/scssVariables.test.js) - would it make sense to combine the test setup?

Reply:
Potentially... I think there is definitely some cleanup and better testing we could be doing for formats for sure. I think we can tackle that in a separate PR though.